kfs-purap/src/main/java/edu/arizona/kfs/module/purap/service/impl/PurapGeneralLedgerServiceImpl.java

```diff
@@ -0,0 +1,231 @@
+package edu.arizona.kfs.module.purap.service.impl;
```

Comment:
I don't see any code in this class for KITT-592 (non-check non-ach), only the changes for KFSI-5964. Where is it? Did you put it elsewhere?